teuthology/provision.py

```diff
@@ -195,6 +200,168 @@ def __del__(self):
         self.remove_config()
 
 
+class ProvisionOpenStack(OpenStack):
```

Comment:
Well, ``init_user_data()`` is "building a config" and ``__del__()`` is "removing a config"